.gitignore

```diff
@@ -83,6 +84,9 @@ docs/**/*.md
 *.cmx
 *.cmo
 .history
-docs/org/README.org
+
 # Binary files (produced by `make check`)
-examples/milestone/HelloWorld/HelloWorld
+examples/**/*
+examples/**/!*.*
+test/**/*
```

Comment:
Question: Should this be `tests/` as the `test/` directory contains the Haskell code, the Juvix code is in `tests/` ?

Reply:
Good catch! it's tests indeed